lib/zuora/objects/product_rate_plan.rb

```diff
@@ -4,11 +4,12 @@ class ProductRatePlan < Base
     has_many :product_rate_plan_charges
 
     validates_length_of :description, :maximum => 500, :allow_nil => true
-    validates_datetime_of :effective_start_date, :effective_end_date
+    # validates_datetime_of :effective_start_date, :effective_end_date
```

Comment:
Why is this commented out?

Reply:
Resolved here [04f281f](https://github.com/Sage/zuora/pull/18/commits/04f281f9a0a0b07961e1b1f4ebd5465beb8c27bc)